docs/docs/dev_docs/contracts/portals/main.md

```diff
@@ -71,10 +75,13 @@ When sending a message from L2 to L1 we don't need to pass recipient, deadline,
 Access control on the L1 portal contract is essential to prevent consumption of messages sent from the wrong L2 contract.
 :::
 
-As earlier, we can use a token bridge as an example. In this case, we are burning tokens on L2 and sending a message to the portal to free them on L1.
+As earlier, we can use a token bridge as an example. In this case, we are burning tokens on L2 and sending a message to the portal to free them on L1. The burn happens privately, so it doesn't leak the burn amount on Aztec.
```

Comment:
The burn amount will be leaked on Aztec because the message that is making its way to L1 is including it, and for the burn specifically it is also updating the total_supply to keep that consistent. So you could see the decrease in total supply and know the amount.
You are essentially hiding who is doing the burn.